security/storage/aws/public_buckets/aws_public_buckets.pt

```diff
@@ -187,19 +187,16 @@ end
 
 
 escalation "report_open_buckets" do
+  automatic true
+  label "Send Mail"
+  description "Sends incidents email"
   email $param_email
 end
 
 escalation "slack_open_buckets" do
-  request_approval  do
-    label "Approve to slack the open buckets"
-    description "Send the message to slack the open buckets"
-    parameter "approval_reason" do
-      type "string"
-      label "Reason for approval"
-      description "Explain why you are approving the action"
-    end
-  end
+  automatic false
+  label "Slack Open Buckets"
+  description "Approval to slack all selected open buckets"
   run "send_slack", data, rs_project_name, rs_project_id, $param_slack_channel, $param_slack_webhook_cred
 end
```

Comment:
Sending a slack message is inconsistent with our other policies.  Can we remove the param, CWF and escalation for this?  Readme and changelog would need to reflect this change.  thx

Reply:
Done!